### ISSUE TITLE
Raw bytecode debugging

### ISSUE THREAD

**Issue by SoniEx2** [pkulchenko/MobDebug#45]
Would it be possible to debug raw bytecode with string.dump and a count hook?

**Comment by pkulchenko** [comment]
@SoniEx2, somebody would have to try that, but there wouldn't be anything to step through, as you'd still need the source code to work with in the debugger, right?

**Comment by SoniEx2** [comment]
you can step through instructions.

**Comment by SoniEx2** [comment]
I'm not sure how feasible it is to have luac -l -l as part of the debugger. but it'd be kinda nice imo.

**Comment by pkulchenko** [comment]
I agree, this sounds like an interesting idea, which should be quite doable using the source map support in mobdebug, but not something I have short term plans for.
